### DISCUSSION TITLE
Turn on and off Camera Layer doesn’t work

### DISCUSSION THREAD

**Discussion by invishane** [keiwando/toonsquid#535] [General]
Hey! Sorry if I’m missing something here but when I hide the camera layer it just hides the guidelines and handles - the camera layer will still function. Sometimes I want to view my whole scene without the camera layer but hiding it doesn’t work for that. 

Thanks!

**Comment by ll-ghin** [comment]
Hi invishane, you can go to Settings > General > Follow the camera layer during playback and toggle the switch.  Try that and let me know if it works and performs the way you want it to.

**Reply by invishane** [reply]
ok i missed that! That's why i prefaced my comment, now i feel stupid haha but yes that worked. Thank you so much for your help! Can't convey my appreciation enough

**Reply by ll-ghin** [reply]
Glad to hear it! I love Toonsquid so I hope you can enjoy it more with this setting.
